Cargo.toml

```diff
@@ -92,9 +95,12 @@ owo-colors = "3.5.0"
 parking_lot = "0.12.0"
 percent-encoding = "2.1.0"
 pin-project = "1.0.12"
+pin-project-lite = "*"
```

Comment:
We should not be using "*" dependencies

Reply:
Fixed

Reply:
There still seem to be some in the pushed version.